tests/user/actions/set_present.yml

```diff
@@ -0,0 +1,13 @@
+---
+action: user.set_present
+payload:
+  meeting_id: 1
+
+cases:
+- name: Without settings
+  is_allowed: false
+
+- name: With settings
+  db:
+    meeting/1/users_allow_self_set_present: true
```

Comment:
This should be checked by the backend, right? This is not a permission check.

You should check all user actions against the anonymous.

Reply:
As I think about it. We should make clear, what the permission service checks. My current thought:

- permissions: yes, it is the permission service
- group assignees (e.g. for motion_comments): yes, since access is defined with the groups
- settings: no, this is business logic and does not have something todo with access.
- Maybe there are more things you check here @oscar. I think we should open an issue and make this clear.

Reply:
I don't understand what you are saying. Maybe a new issue would be a good idea.

Reply:
https://github.com/OpenSlides/openslides-permission-service/issues/167